sda-commons-server-testing/README.md

```diff
@@ -16,6 +16,11 @@ In case you want to use JUnit 5 you also have to activate it in your build.gradl
   test {
     useJUnitPlatform()
   }
+
+  // in case you use the integrationTest plugin:
+  integrationTest {
+    useJUnitPlatform()
```

Comment:
we might want to use inheritance here
let me check the syntax

Reply:
configurations {
    integrationTest.extendsFrom test   
}

Reply:
@ericrath what is the advantage of your proposal over the suggestion made by @christopher-cudennec

Reply:
it makes sure, that our integration tests dont differ from the junit base.
Especially if we add functionality to test{}

Reply:
I think I would go with your suggestion if there was more configuration to the test plugin. 
For now it's the only thing that needs to be configured, so I would stick with the current version.